keyboards/creatkeebs/thera/info.json

```diff
@@ -0,0 +1,54 @@
+{
```

Comment:
This is a via json, and as such does not belong in this repo.

Reply:
Sorry about it, I have changed it now!